### PR TITLE
Comments, abstractization

### PR DESCRIPTION
**Pull request by razhakin**

### CODE REVIEW

**Review by KaplanL — APPROVED** [review]

File: Constants.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
     public static final int shooter2 = 7;
     public static final int shooter3 = 8;
     public static final int shooter4 = 9;
+    public static final double shootingTime = 0.5;
```
Comment (Statej) [inline Constants.java:30]: This specifies for how long will the shooter spin? I would say that for anything which is not 100% clear we should include a short comment saying what it is. e.g. // time specifying how long will the shooter wheels spin

File: Constants.java [hunk 2]
```diff
@@ -41,18 +42,24 @@
     //Bosch motor speed
```
Comment (Statej) [inline Constants.java:42]: In the above line 40 please add comment saying when is this timeout used

File: commands/ShootCommand.java [hunk 3]
```diff
@@ -8,39 +8,50 @@
...
 import edu.wpi.first.wpilibj.command.TimedCommand;
+import frc.robot.Constants;
 import frc.robot.Robot;
 
 /**
- * Add your docs here.
+ * Command for shooting from the shooter. Timed - will execute only for some time, then terminate.
  */
 public class ShootCommand extends TimedCommand {
   /**
```
Comment (Statej) [inline commands/ShootCommand.java:18]: missing docs

File: subsystems/RotateSubsystem.java [hunk 4]
```diff
@@ -57,7 +57,7 @@ protected double returnPIDInput() {
    */
   @Override
   protected void usePIDOutput(double output) {
-    Robot.driveExecutor.setZ(output * 0.5);
+    Robot.driveExecutor.setZ(output * Constants.gyroScalingFactor);
   }
 
   /**
```
Comment (Statej) [inline subsystems/RotateSubsystem.java:63]: rewrite the below sentence please

**Review by Statej — CHANGES_REQUESTED** [review]

**Pull request closed without merging**